csharp/private/common.bzl

```diff
@@ -16,6 +16,9 @@ def is_debug(ctx):
     # equivalent to dbg right now but might want to support this in the future.
     return ctx.var["COMPILATION_MODE"] != "opt"
 
+def use_highentropyva(tfm):
+    return tfm not in ["net20", "net40"]
```

Comment:
I figure this list is really unlikely to change, which is why I didn't make it a lookup table like the rest.
But maybe Microsoft will release reference assemblies for `net35`?
If you think this should be a lookup table like the rest (so that we have to explicitly choose) I can do that.